includes/settings.php

```diff
@@ -750,7 +750,7 @@ public static function _enqueue_assets() {
 			$box_data .= '</div>';
 			$box_data .= '<script>';
 			$box_data .= '$( "#ebutton" ).click( function () {';
-			$box_data .= '$.post( "' . $data . '", $( "#formdata" ).serialize(), function( data ) {';
+			$box_data .= '$.post( "' . $mailin . '", $( "#formdata" ).serialize(), function( data ) {';
```

Comment:
@rohittm `$mailin` is an object you need to replace that with the page URL `http://example.url/`, also  `$( "#formdata" ).serialize()` was an example you should add an id to the form in your case you should replace it with ``$( "form.menu-icons-submit-mail").serialize()` -- this will take all the fields inside the form and serialize them for ajax posting, they will be posted to the url where this page is executed.

Then the condition from line `723` will trigger:
```
if ( ! empty( $_POST['menu_icons_mail'] ) ) {
    require( plugin_dir_path( __DIR__ ) . 'mailin.php' );
    $user_info = get_userdata( 1 );
    $mailin    = new Mailin( 'https://api.sendinblue.com/v2.0', 'cHW5sxZnzE7mhaYb' );
    $data      = array(
       'email'           => $_POST['menu_icons_mail'],
 	'attributes'      => array(
 	    'NAME'    => $user_info->first_name,
 	    'SURNAME' => $user_info->last_name,
 	),
 	'blacklisted'     => 0,
 	'listid'          => array( 51 ),
 	'blacklisted_sms' => 0,
    );
    $status    = $mailin->create_update_user( $data );
    if ( $status['code'] == 'success' ) {
        update_option( 'menu_icons_subscribe', true );
    }
}
```
This is the workflow with AJAX.
You just do the same thing as before where the page would `POST` to itself and this would trigger a page reload, now you `POST` to the page and invoke it but don't leave the active page thus don't reload.

Let me know if you understand or have any more questions.